contractcourt/commit_sweep_resolver_test.go

```diff
@@ -273,14 +274,16 @@ func TestCommitSweepResolverLocalBreach(t *testing.T) {
 	// already swept.
 	ctx.sweeper.sweepErr = sweep.ErrRemoteSpend
 
-	report := ctx.resolver.report()
-	if !reflect.DeepEqual(report, &ContractReport{
+	gotReport := ctx.resolver.report()
+	wantReport := ContractReport{
```

Comment:
Mostly we would use `report` and `expectedReport`, if you want to stay in line with the usual naming (not a hard standard).